<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Access and edit gRNA sequences from the GeCKO Library • GeCKO</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha256-916EbMg70RQy9LHiGkXzG8hSg9EdNy97GazNG/aiY1w=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- sticky kit --><script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script><!-- pkgdown --><link href="pkgdown.css" rel="stylesheet">
<script src="pkgdown.js"></script><meta property="og:title" content="Access and edit gRNA sequences from the GeCKO Library">
<meta property="og:description" content="Access and edit gRNA sequences from the GeCKO Library">
<meta property="og:image" content="/logo.png">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-home">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="index.html">GeCKO</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="reference/index.html">Reference</a>
</li>
<li>
  <a href="news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/dbrookeUAB/GeCKO">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><div class="row">
  <div class="contents col-md-9">
<div id="gecko" class="section level1">
<div class="page-header"><h1 class="hasAnchor">
<a href="#gecko" class="anchor"></a>GeCKO <img src="reference/figures/logo.png" align="right" alt="" width="120">
</h1></div>
<p><br><br><!-- badges: start --><a href="https://travis-ci.org/dbrookeUAB/GeCKO"><img src="https://travis-ci.org/dbrookeUAB/GeCKO.svg?branch=master" alt="Build Status"></a><a href="https://travis-ci.org/dbrookeUAB/GeCKO"><img src="https://travis-ci.org/dbrookeUAB/GeCKO.svg?branch=Developmental" alt="Developmental Status"></a><a href="https://github.com/dbrookeUAB/GeCKO"><img src="https://img.shields.io/github/languages/code-size/dbrookeUAB/GeCKO.svg" alt="Code Size"></a> <a href="https://github.com/dbrookeUAB/GeCKO/commits/master"><img src="https://img.shields.io/github/last-commit/dbrookeUAB/GeCKO.svg" alt="Last Commit"></a> <a href="https://codecov.io/gh/dbrookeUAB/GeCKO?branch=master"><img src="https://codecov.io/gh/dbrookeUAB/GeCKO/branch/master/graph/badge.svg" alt="Codecov test coverage"></a> <!-- badges: end --></p>
<p>The goal of GeCKO is to …</p>
<div id="installation" class="section level2">
<h2 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="co"># install.packages("remotes")</span></a>
<a class="sourceLine" id="cb1-2" data-line-number="2">remotes<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/remotes/topics/install_github">install_github</a></span>(<span class="st">"dbrookeUAB/GeCKO"</span>)</a></code></pre></div>
</div>
<div id="basic-usage" class="section level2">
<h2 class="hasAnchor">
<a href="#basic-usage" class="anchor"></a>Basic Usage</h2>
<p>Make sure to cite this paper. <a href="https://science.sciencemag.org/content/343/6166/84">O. Shalem et al., Genome-scale CRISPR-Cas9 knockout screening in human cells. Science (80-. ). 343, 84–87 (2014)</a></p>
<p><a href="http://genome-engineering.org">Zhang Lab Website</a></p>
<div id="loading-grna-sequences-from-the-gecko-library" class="section level3">
<h3 class="hasAnchor">
<a href="#loading-grna-sequences-from-the-gecko-library" class="anchor"></a>Loading gRNA sequences from the GeCKO Library</h3>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(GeCKO)</a>
<a class="sourceLine" id="cb2-2" data-line-number="2">human &lt;-<span class="st"> </span><span class="kw"><a href="reference/get_library.html">get_library</a></span>(<span class="dt">species =</span> <span class="st">"human"</span>) </a>
<a class="sourceLine" id="cb2-3" data-line-number="3"></a>
<a class="sourceLine" id="cb2-4" data-line-number="4"><span class="co"># not run</span></a>
<a class="sourceLine" id="cb2-5" data-line-number="5"><span class="co"># mouse &lt;- get_library(species = "mouse")</span></a>
<a class="sourceLine" id="cb2-6" data-line-number="6"></a>
<a class="sourceLine" id="cb2-7" data-line-number="7">human</a>
<a class="sourceLine" id="cb2-8" data-line-number="8"><span class="co">#&gt;             gene_id          UID                  seq</span></a>
<a class="sourceLine" id="cb2-9" data-line-number="9"><span class="co">#&gt;      1:        A1BG HGLibA_00001 GTCGCTGAGCTCCGATTCGA</span></a>
<a class="sourceLine" id="cb2-10" data-line-number="10"><span class="co">#&gt;      2:        A1BG HGLibA_00002 ACCTGTAGTTGCCGGCGTGC</span></a>
<a class="sourceLine" id="cb2-11" data-line-number="11"><span class="co">#&gt;      3:        A1BG HGLibA_00003 CGTCAGCGTCACATTGGCCA</span></a>
<a class="sourceLine" id="cb2-12" data-line-number="12"><span class="co">#&gt;      4:        A1BG HGLibB_00001 CAATGTGACGCTGACGTGCC</span></a>
<a class="sourceLine" id="cb2-13" data-line-number="13"><span class="co">#&gt;      5:        A1BG HGLibB_00002 TGTCTCCGCAGGTGTCACCT</span></a>
<a class="sourceLine" id="cb2-14" data-line-number="14"><span class="co">#&gt;     ---                                              </span></a>
<a class="sourceLine" id="cb2-15" data-line-number="15"><span class="co">#&gt; 123407: hsa-mir-99a HGLibA_64379 ATCTACGGGTTTATGCCAAT</span></a>
<a class="sourceLine" id="cb2-16" data-line-number="16"><span class="co">#&gt; 123408: hsa-mir-99b HGLibA_64380 CACCCGTAGAACCGACCTTG</span></a>
<a class="sourceLine" id="cb2-17" data-line-number="17"><span class="co">#&gt; 123409: hsa-mir-99b HGLibA_64381 CCCGTAGAACCGACCTTGCG</span></a>
<a class="sourceLine" id="cb2-18" data-line-number="18"><span class="co">#&gt; 123410: hsa-mir-99b HGLibA_64382 CGCACACAAGCTCGTGTCTG</span></a>
<a class="sourceLine" id="cb2-19" data-line-number="19"><span class="co">#&gt; 123411: hsa-mir-99b HGLibA_64383 TCGTGTCTGTGGGTCCGTGT</span></a></code></pre></div>
<div id="looking-for-a-specific-gene" class="section level4">
<h4 class="hasAnchor">
<a href="#looking-for-a-specific-gene" class="anchor"></a>Looking for a specific gene</h4>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" data-line-number="1"><span class="co"># using data.table notation DT[i,j,by]</span></a>
<a class="sourceLine" id="cb3-2" data-line-number="2">single_gene&lt;-<span class="st"> </span>human[gene_id<span class="op">==</span><span class="st">"GAPDH"</span>]</a>
<a class="sourceLine" id="cb3-3" data-line-number="3">single_gene</a>
<a class="sourceLine" id="cb3-4" data-line-number="4"><span class="co">#&gt;    gene_id          UID                  seq</span></a>
<a class="sourceLine" id="cb3-5" data-line-number="5"><span class="co">#&gt; 1:   GAPDH HGLibA_18613 TTCCTCACCTGATGATCTTG</span></a>
<a class="sourceLine" id="cb3-6" data-line-number="6"><span class="co">#&gt; 2:   GAPDH HGLibA_18614 TCTGCTGTAGGCTCATTTGC</span></a>
<a class="sourceLine" id="cb3-7" data-line-number="7"><span class="co">#&gt; 3:   GAPDH HGLibA_18615 TTCCACTCACTCCTGGAAGA</span></a>
<a class="sourceLine" id="cb3-8" data-line-number="8"><span class="co">#&gt; 4:   GAPDH HGLibB_18588 ACTCACCCCAGCCTTCTCCA</span></a>
<a class="sourceLine" id="cb3-9" data-line-number="9"><span class="co">#&gt; 5:   GAPDH HGLibB_18589 CACTGGCGTCTTCACCACCA</span></a>
<a class="sourceLine" id="cb3-10" data-line-number="10"><span class="co">#&gt; 6:   GAPDH HGLibB_18590 TTCCAATATGATTCCACCCA</span></a></code></pre></div>
</div>
<div id="retrieving-multiple-genes" class="section level4">
<h4 class="hasAnchor">
<a href="#retrieving-multiple-genes" class="anchor"></a>Retrieving Multiple Genes</h4>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1">many_genes&lt;-human[gene_id <span class="op">%in%</span><span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"GAPDH"</span>,<span class="st">"TP53"</span>, <span class="st">"CCND1"</span>)]</a>
<a class="sourceLine" id="cb4-2" data-line-number="2">man_genes</a>
<a class="sourceLine" id="cb4-3" data-line-number="3"><span class="co">#&gt;     gene_id          UID                  seq</span></a>
<a class="sourceLine" id="cb4-4" data-line-number="4"><span class="co">#&gt;  1:   CCND1 HGLibA_08071 CGAAGGTCTGCGCGTGTTTG</span></a>
<a class="sourceLine" id="cb4-5" data-line-number="5"><span class="co">#&gt;  2:   CCND1 HGLibA_08072 CTTGCACACCCACCGACGTG</span></a>
<a class="sourceLine" id="cb4-6" data-line-number="6"><span class="co">#&gt;  3:   CCND1 HGLibA_08073 GATGTCCACGTCCCGCACGT</span></a>
<a class="sourceLine" id="cb4-7" data-line-number="7"><span class="co">#&gt;  4:   CCND1 HGLibB_08064 GTTCGTGGCCTCTAAGATGA</span></a>
<a class="sourceLine" id="cb4-8" data-line-number="8"><span class="co">#&gt;  5:   CCND1 HGLibB_08065 TGTTTGTTCTCCTCCGCCTC</span></a>
<a class="sourceLine" id="cb4-9" data-line-number="9"><span class="co">#&gt;  6:   CCND1 HGLibB_08066 GAAGCGTGTGAGGCGGTAGT</span></a>
<a class="sourceLine" id="cb4-10" data-line-number="10"><span class="co">#&gt;  7:   GAPDH HGLibA_18613 TTCCTCACCTGATGATCTTG</span></a>
<a class="sourceLine" id="cb4-11" data-line-number="11"><span class="co">#&gt;  8:   GAPDH HGLibA_18614 TCTGCTGTAGGCTCATTTGC</span></a>
<a class="sourceLine" id="cb4-12" data-line-number="12"><span class="co">#&gt;  9:   GAPDH HGLibA_18615 TTCCACTCACTCCTGGAAGA</span></a>
<a class="sourceLine" id="cb4-13" data-line-number="13"><span class="co">#&gt; 10:   GAPDH HGLibB_18588 ACTCACCCCAGCCTTCTCCA</span></a>
<a class="sourceLine" id="cb4-14" data-line-number="14"><span class="co">#&gt; 11:   GAPDH HGLibB_18589 CACTGGCGTCTTCACCACCA</span></a>
<a class="sourceLine" id="cb4-15" data-line-number="15"><span class="co">#&gt; 12:   GAPDH HGLibB_18590 TTCCAATATGATTCCACCCA</span></a>
<a class="sourceLine" id="cb4-16" data-line-number="16"><span class="co">#&gt; 13:    TP53 HGLibA_51047 CCGGTTCATGCCGCCCATGC</span></a>
<a class="sourceLine" id="cb4-17" data-line-number="17"><span class="co">#&gt; 14:    TP53 HGLibA_51048 CGCTATCTGAGCAGCGCTCA</span></a>
<a class="sourceLine" id="cb4-18" data-line-number="18"><span class="co">#&gt; 15:    TP53 HGLibA_51049 CCCCGGACGATATTGAACAA</span></a>
<a class="sourceLine" id="cb4-19" data-line-number="19"><span class="co">#&gt; 16:    TP53 HGLibB_50984 GAGCGCTGCTCAGATAGCGA</span></a>
<a class="sourceLine" id="cb4-20" data-line-number="20"><span class="co">#&gt; 17:    TP53 HGLibB_50985 CCCCTTGCCGTCCCAAGCAA</span></a>
<a class="sourceLine" id="cb4-21" data-line-number="21"><span class="co">#&gt; 18:    TP53 HGLibB_50986 ACTTCCTGAAAACAACGTTC</span></a></code></pre></div>
</div>
<div id="making-grna-oligos-for-lenticrispr-v2-or-lentiguide-puro" class="section level4">
<h4 class="hasAnchor">
<a href="#making-grna-oligos-for-lenticrispr-v2-or-lentiguide-puro" class="anchor"></a>Making gRNA Oligos for <a href="https://www.addgene.org/52961/">LentiCRISPR v2</a> or <a href="https://www.addgene.org/52963/">LentiGuide-Puro</a>
</h4>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" data-line-number="1"><span class="co"># simple example</span></a>
<a class="sourceLine" id="cb5-2" data-line-number="2"><span class="kw"><a href="reference/make_gRNAs.html">make_gRNAs</a></span>(<span class="dt">seqs =</span> <span class="st">"TGTCAACGGGCGGCCACTGC"</span>,<span class="dt">gene =</span> <span class="st">"PAX6"</span>)</a>
<a class="sourceLine" id="cb5-3" data-line-number="3"><span class="co">#&gt;    gene_id  gRNA                 seqs     idt_id_F</span></a>
<a class="sourceLine" id="cb5-4" data-line-number="4"><span class="co">#&gt; 1:    PAX6 gRNA1 TGTCAACGGGCGGCCACTGC F-PAX6_gRNA1</span></a>
<a class="sourceLine" id="cb5-5" data-line-number="5"><span class="co">#&gt;             oligo-1 (5'-&gt;3')     idt_id_R          oligo-2 (5'-&gt;3')</span></a>
<a class="sourceLine" id="cb5-6" data-line-number="6"><span class="co">#&gt; 1: CACCGTGTCAACGGGCGGCCACTGC R-PAX6_gRNA1 AAACGCAGTGGCCGCCCGTTGACAC</span></a></code></pre></div>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1"><span class="co"># using data.tables directly for a single gene</span></a>
<a class="sourceLine" id="cb6-2" data-line-number="2"><span class="kw"><a href="reference/make_gRNAs.html">make_gRNAs</a></span>(single_gene[,seq], <span class="dt">gene =</span> single_gene[,gene_id])</a>
<a class="sourceLine" id="cb6-3" data-line-number="3"><span class="co">#&gt;    gene_id  gRNA                 seqs      idt_id_F</span></a>
<a class="sourceLine" id="cb6-4" data-line-number="4"><span class="co">#&gt; 1:   GAPDH gRNA1 TTCCTCACCTGATGATCTTG F-GAPDH_gRNA1</span></a>
<a class="sourceLine" id="cb6-5" data-line-number="5"><span class="co">#&gt; 2:   GAPDH gRNA2 TCTGCTGTAGGCTCATTTGC F-GAPDH_gRNA2</span></a>
<a class="sourceLine" id="cb6-6" data-line-number="6"><span class="co">#&gt; 3:   GAPDH gRNA3 TTCCACTCACTCCTGGAAGA F-GAPDH_gRNA3</span></a>
<a class="sourceLine" id="cb6-7" data-line-number="7"><span class="co">#&gt; 4:   GAPDH gRNA4 ACTCACCCCAGCCTTCTCCA F-GAPDH_gRNA4</span></a>
<a class="sourceLine" id="cb6-8" data-line-number="8"><span class="co">#&gt; 5:   GAPDH gRNA5 CACTGGCGTCTTCACCACCA F-GAPDH_gRNA5</span></a>
<a class="sourceLine" id="cb6-9" data-line-number="9"><span class="co">#&gt; 6:   GAPDH gRNA6 TTCCAATATGATTCCACCCA F-GAPDH_gRNA6</span></a>
<a class="sourceLine" id="cb6-10" data-line-number="10"><span class="co">#&gt;             oligo-1 (5'-&gt;3')      idt_id_R          oligo-2 (5'-&gt;3')</span></a>
<a class="sourceLine" id="cb6-11" data-line-number="11"><span class="co">#&gt; 1: CACCGTTCCTCACCTGATGATCTTG R-GAPDH_gRNA1 AAACCAAGATCATCAGGTGAGGAAC</span></a>
<a class="sourceLine" id="cb6-12" data-line-number="12"><span class="co">#&gt; 2: CACCGTCTGCTGTAGGCTCATTTGC R-GAPDH_gRNA2 AAACGCAAATGAGCCTACAGCAGAC</span></a>
<a class="sourceLine" id="cb6-13" data-line-number="13"><span class="co">#&gt; 3: CACCGTTCCACTCACTCCTGGAAGA R-GAPDH_gRNA3 AAACTCTTCCAGGAGTGAGTGGAAC</span></a>
<a class="sourceLine" id="cb6-14" data-line-number="14"><span class="co">#&gt; 4: CACCGACTCACCCCAGCCTTCTCCA R-GAPDH_gRNA4 AAACTGGAGAAGGCTGGGGTGAGTC</span></a>
<a class="sourceLine" id="cb6-15" data-line-number="15"><span class="co">#&gt; 5: CACCGCACTGGCGTCTTCACCACCA R-GAPDH_gRNA5 AAACTGGTGGTGAAGACGCCAGTGC</span></a>
<a class="sourceLine" id="cb6-16" data-line-number="16"><span class="co">#&gt; 6: CACCGTTCCAATATGATTCCACCCA R-GAPDH_gRNA6 AAACTGGGTGGAATCATATTGGAAC</span></a></code></pre></div>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb7-1" data-line-number="1"><span class="co"># using data.tables directly for a multiple genes</span></a>
<a class="sourceLine" id="cb7-2" data-line-number="2"><span class="kw"><a href="reference/make_gRNAs.html">make_gRNAs</a></span>(many_genes[,seq], <span class="dt">gene =</span> many_genes[,gene_id])</a>
<a class="sourceLine" id="cb7-3" data-line-number="3"><span class="co">#&gt;     gene_id  gRNA                 seqs      idt_id_F</span></a>
<a class="sourceLine" id="cb7-4" data-line-number="4"><span class="co">#&gt;  1:   CCND1 gRNA1 CGAAGGTCTGCGCGTGTTTG F-CCND1_gRNA1</span></a>
<a class="sourceLine" id="cb7-5" data-line-number="5"><span class="co">#&gt;  2:   CCND1 gRNA2 CTTGCACACCCACCGACGTG F-CCND1_gRNA2</span></a>
<a class="sourceLine" id="cb7-6" data-line-number="6"><span class="co">#&gt;  3:   CCND1 gRNA3 GATGTCCACGTCCCGCACGT F-CCND1_gRNA3</span></a>
<a class="sourceLine" id="cb7-7" data-line-number="7"><span class="co">#&gt;  4:   CCND1 gRNA4 GTTCGTGGCCTCTAAGATGA F-CCND1_gRNA4</span></a>
<a class="sourceLine" id="cb7-8" data-line-number="8"><span class="co">#&gt;  5:   CCND1 gRNA5 TGTTTGTTCTCCTCCGCCTC F-CCND1_gRNA5</span></a>
<a class="sourceLine" id="cb7-9" data-line-number="9"><span class="co">#&gt;  6:   CCND1 gRNA6 GAAGCGTGTGAGGCGGTAGT F-CCND1_gRNA6</span></a>
<a class="sourceLine" id="cb7-10" data-line-number="10"><span class="co">#&gt;  7:   GAPDH gRNA1 TTCCTCACCTGATGATCTTG F-GAPDH_gRNA1</span></a>
<a class="sourceLine" id="cb7-11" data-line-number="11"><span class="co">#&gt;  8:   GAPDH gRNA2 TCTGCTGTAGGCTCATTTGC F-GAPDH_gRNA2</span></a>
<a class="sourceLine" id="cb7-12" data-line-number="12"><span class="co">#&gt;  9:   GAPDH gRNA3 TTCCACTCACTCCTGGAAGA F-GAPDH_gRNA3</span></a>
<a class="sourceLine" id="cb7-13" data-line-number="13"><span class="co">#&gt; 10:   GAPDH gRNA4 ACTCACCCCAGCCTTCTCCA F-GAPDH_gRNA4</span></a>
<a class="sourceLine" id="cb7-14" data-line-number="14"><span class="co">#&gt; 11:   GAPDH gRNA5 CACTGGCGTCTTCACCACCA F-GAPDH_gRNA5</span></a>
<a class="sourceLine" id="cb7-15" data-line-number="15"><span class="co">#&gt; 12:   GAPDH gRNA6 TTCCAATATGATTCCACCCA F-GAPDH_gRNA6</span></a>
<a class="sourceLine" id="cb7-16" data-line-number="16"><span class="co">#&gt; 13:    TP53 gRNA1 CCGGTTCATGCCGCCCATGC  F-TP53_gRNA1</span></a>
<a class="sourceLine" id="cb7-17" data-line-number="17"><span class="co">#&gt; 14:    TP53 gRNA2 CGCTATCTGAGCAGCGCTCA  F-TP53_gRNA2</span></a>
<a class="sourceLine" id="cb7-18" data-line-number="18"><span class="co">#&gt; 15:    TP53 gRNA3 CCCCGGACGATATTGAACAA  F-TP53_gRNA3</span></a>
<a class="sourceLine" id="cb7-19" data-line-number="19"><span class="co">#&gt; 16:    TP53 gRNA4 GAGCGCTGCTCAGATAGCGA  F-TP53_gRNA4</span></a>
<a class="sourceLine" id="cb7-20" data-line-number="20"><span class="co">#&gt; 17:    TP53 gRNA5 CCCCTTGCCGTCCCAAGCAA  F-TP53_gRNA5</span></a>
<a class="sourceLine" id="cb7-21" data-line-number="21"><span class="co">#&gt; 18:    TP53 gRNA6 ACTTCCTGAAAACAACGTTC  F-TP53_gRNA6</span></a>
<a class="sourceLine" id="cb7-22" data-line-number="22"><span class="co">#&gt;              oligo-1 (5'-&gt;3')      idt_id_R          oligo-2 (5'-&gt;3')</span></a>
<a class="sourceLine" id="cb7-23" data-line-number="23"><span class="co">#&gt;  1: CACCGCGAAGGTCTGCGCGTGTTTG R-CCND1_gRNA1 AAACCAAACACGCGCAGACCTTCGC</span></a>
<a class="sourceLine" id="cb7-24" data-line-number="24"><span class="co">#&gt;  2: CACCGACTCACCCCAGCCTTCTCCA R-CCND1_gRNA2 AAACTGGAGAAGGCTGGGGTGAGTC</span></a>
<a class="sourceLine" id="cb7-25" data-line-number="25"><span class="co">#&gt;  3: CACCGCACTGGCGTCTTCACCACCA R-CCND1_gRNA3 AAACTGGTGGTGAAGACGCCAGTGC</span></a>
<a class="sourceLine" id="cb7-26" data-line-number="26"><span class="co">#&gt;  4: CACCGTTCCAATATGATTCCACCCA R-CCND1_gRNA4 AAACTGGGTGGAATCATATTGGAAC</span></a>
<a class="sourceLine" id="cb7-27" data-line-number="27"><span class="co">#&gt;  5: CACCGCCGGTTCATGCCGCCCATGC R-CCND1_gRNA5 AAACGCATGGGCGGCATGAACCGGC</span></a>
<a class="sourceLine" id="cb7-28" data-line-number="28"><span class="co">#&gt;  6: CACCGCGCTATCTGAGCAGCGCTCA R-CCND1_gRNA6 AAACTGAGCGCTGCTCAGATAGCGC</span></a>
<a class="sourceLine" id="cb7-29" data-line-number="29"><span class="co">#&gt;  7: CACCGCCCCGGACGATATTGAACAA R-GAPDH_gRNA1 AAACTTGTTCAATATCGTCCGGGGC</span></a>
<a class="sourceLine" id="cb7-30" data-line-number="30"><span class="co">#&gt;  8: CACCGGAGCGCTGCTCAGATAGCGA R-GAPDH_gRNA2 AAACTCGCTATCTGAGCAGCGCTCC</span></a>
<a class="sourceLine" id="cb7-31" data-line-number="31"><span class="co">#&gt;  9: CACCGCCCCTTGCCGTCCCAAGCAA R-GAPDH_gRNA3 AAACTTGCTTGGGACGGCAAGGGGC</span></a>
<a class="sourceLine" id="cb7-32" data-line-number="32"><span class="co">#&gt; 10: CACCGACTTCCTGAAAACAACGTTC R-GAPDH_gRNA4 AAACGAACGTTGTTTTCAGGAAGTC</span></a>
<a class="sourceLine" id="cb7-33" data-line-number="33"><span class="co">#&gt; 11: CACCGCTTGCACACCCACCGACGTG R-GAPDH_gRNA5 AAACCACGTCGGTGGGTGTGCAAGC</span></a>
<a class="sourceLine" id="cb7-34" data-line-number="34"><span class="co">#&gt; 12: CACCGGATGTCCACGTCCCGCACGT R-GAPDH_gRNA6 AAACACGTGCGGGACGTGGACATCC</span></a>
<a class="sourceLine" id="cb7-35" data-line-number="35"><span class="co">#&gt; 13: CACCGGTTCGTGGCCTCTAAGATGA  R-TP53_gRNA1 AAACTCATCTTAGAGGCCACGAACC</span></a>
<a class="sourceLine" id="cb7-36" data-line-number="36"><span class="co">#&gt; 14: CACCGTGTTTGTTCTCCTCCGCCTC  R-TP53_gRNA2 AAACGAGGCGGAGGAGAACAAACAC</span></a>
<a class="sourceLine" id="cb7-37" data-line-number="37"><span class="co">#&gt; 15: CACCGGAAGCGTGTGAGGCGGTAGT  R-TP53_gRNA3 AAACACTACCGCCTCACACGCTTCC</span></a>
<a class="sourceLine" id="cb7-38" data-line-number="38"><span class="co">#&gt; 16: CACCGTTCCTCACCTGATGATCTTG  R-TP53_gRNA4 AAACCAAGATCATCAGGTGAGGAAC</span></a>
<a class="sourceLine" id="cb7-39" data-line-number="39"><span class="co">#&gt; 17: CACCGTCTGCTGTAGGCTCATTTGC  R-TP53_gRNA5 AAACGCAAATGAGCCTACAGCAGAC</span></a>
<a class="sourceLine" id="cb7-40" data-line-number="40"><span class="co">#&gt; 18: CACCGTTCCACTCACTCCTGGAAGA  R-TP53_gRNA6 AAACTCTTCCAGGAGTGAGTGGAAC</span></a></code></pre></div>
</div>
</div>
</div>
<div id="using-custom-5-and-3-sequences-for-grna-oligos" class="section level2">
<h2 class="hasAnchor">
<a href="#using-custom-5-and-3-sequences-for-grna-oligos" class="anchor"></a>Using custom 5’ and 3’ sequences for gRNA oligos</h2>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" data-line-number="1"><span class="co"># add2 uses a list input</span></a>
<a class="sourceLine" id="cb8-2" data-line-number="2"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(GeCKO)</a>
<a class="sourceLine" id="cb8-3" data-line-number="3"><span class="kw"><a href="reference/make_gRNAs.html">make_gRNAs</a></span>(<span class="dt">seqs =</span> <span class="st">"TGTCAACGGGCGGCCACTGC"</span>,<span class="dt">gene =</span> <span class="st">"PAX6"</span>,</a>
<a class="sourceLine" id="cb8-4" data-line-number="4">            <span class="dt">add2 =</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>(</a>
<a class="sourceLine" id="cb8-5" data-line-number="5">              <span class="dt">seq1_5p =</span> <span class="st">"new"</span>,</a>
<a class="sourceLine" id="cb8-6" data-line-number="6">              <span class="dt">seq1_3p =</span> <span class="st">"ends"</span>,</a>
<a class="sourceLine" id="cb8-7" data-line-number="7">              <span class="dt">seq2_5p =</span> <span class="st">"for"</span>, </a>
<a class="sourceLine" id="cb8-8" data-line-number="8">              <span class="dt">seq2_3p =</span> <span class="st">"both"</span>)</a>
<a class="sourceLine" id="cb8-9" data-line-number="9">            )</a>
<a class="sourceLine" id="cb8-10" data-line-number="10"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb8-11" data-line-number="11"><span class="co">#&gt;    gene_id  gRNA                 seqs     idt_id_F</span></a>
<a class="sourceLine" id="cb8-12" data-line-number="12"><span class="co">#&gt; 1:    PAX6 gRNA1 TGTCAACGGGCGGCCACTGC F-PAX6_gRNA1</span></a>
<a class="sourceLine" id="cb8-13" data-line-number="13"><span class="co">#&gt;               oligo-1 (5'-&gt;3')     idt_id_R            oligo-2 (5'-&gt;3')</span></a>
<a class="sourceLine" id="cb8-14" data-line-number="14"><span class="co">#&gt; 1: newTGTCAACGGGCGGCCACTGCends R-PAX6_gRNA1 forGCAGTGGCCGCCCGTTGACAboth</span></a></code></pre></div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
    <div class="links">
<h2>Links</h2>
<ul class="list-unstyled">
<li>Browse source code at <br><a href="https://github.com/dbrookeUAB/GeCKO">https://​github.com/​dbrookeUAB/​GeCKO</a>
</li>
<li>Report a bug at <br><a href="https://github.com/dbrookeUAB/GeCKO/issues">https://​github.com/​dbrookeUAB/​GeCKO/​issues</a>
</li>
</ul>
</div>
<div class="license">
<h2>License</h2>
<ul class="list-unstyled">
<li>GPL (&gt;= 2)</li>
</ul>
</div>
<div class="developers">
<h2>Developers</h2>
<ul class="list-unstyled">
<li>Dewey Brooke <br><small class="roles"> Author, maintainer </small> <a href="https://orcid.org/0000-0003-4290-3809" target="orcid.widget"><img src="https://members.orcid.org/sites/default/files/vector_iD_icon.svg" class="orcid" alt="ORCID" height="16"></a> </li>
</ul>
</div>

  </div>
</div>

      <footer><div class="copyright">
  <p>Developed by Dewey Brooke.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
</div>

  

  </body>
</html>
